### DISCUSSION TITLE
Create catalog schema for static files

### DISCUSSION THREAD

**Discussion by apreucil** [NOAA-GFDL/CatalogBuilder#49] [Ideas]
Statics files are of great importance and use for the analysis community, and I think users would be happy to have a tool that gathers static file information into a simple and navigable catalog. The schema for the static files does not necessarily need to be different than the model schema, but I imagine the way the catalog is created would need to be different since there the internal file structure of the static file is inherently different than a model output file.

**Comment by aradhakrishnanGFDL** [comment]
I started a discussion in the MDTF repository which may be of interest. Please chime in with ideas.

https://github.com/NOAA-GFDL/MDTF-diagnostics/discussions/686
